scripts/insertComments/comments.txt

```diff
@@ -0,0 +1,4 @@
+尹麗喬，哈佛費正清中國研究中心研究員
+
+知識就是力量。台美應加強情資交流合作，以共同對抗網路間諜及假新聞；台灣應成為「第六眼」。
+Scientia potentia est (knowledge is power). U.S.-Taiwan should deepen intelligence exchanges to combat cyber espionage and fake news; Taiwan should become the sixth eye.
```

Comment:
If this is meant to be an example, it should have comments at the top that denote that it is as such